packages/cra-template/template/src/service-worker.js

```diff
@@ -0,0 +1,59 @@
+// eslint-disable no-restricted-globals
```

Comment:
I believe this needs to use the  `/* eslint-disable no-restricted-globals */` syntax, instead of the other JS comment syntax, to apply to the entire file.

Reply:
Ah yes. Forgot about that difference between next-line and file 😛